### DISCUSSION TITLE
Delete extra step in Payment Request Archiving flow

### DISCUSSION THREAD

**Discussion by Zaxounette** [btcpayserver/btcpayserver#3134] [Ideas / Feature Requests]
### The issue:

Currently, when archiving a Payment Request, we are taken into that specific Payment Request the same way we would be if we chose to edit it.

See gif below for current flow:
![ABrQ3wRn3s](https://user-images.githubusercontent.com/51208677/142738839-f2081775-c98a-453e-afb7-1efb84e53292.gif)

### Ideal flow

The ideal flow would simply remove the Payment Request from the list view upon selecting "Archive", with potentially a "are you sure?" box (though archiving is not a permanent action, so not 100% necessary) and the confirmation message would appear in the list view instead of the Payment Request-edit view.

This would remove an extra step (and click) for the same result.

Thoughts?

## BTCPay Environment:
- BTCPay Server Version: v1.3.6.0
- Deployment Method: Lunanode
- Browser:  Chrome & Firefix

**Comment by bolatovumar** [comment]
PR here: https://github.com/btcpayserver/btcpayserver/pull/3142
